examples/ui.apps/src/main/content/jcr_root/apps/forms-components-examples/components/form/title/.content.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
we need to have sling model of title as this is where we'll handle localization. Then I guess, we'll have to copy the sightly too.
@rismehta